### DISCUSSION TITLE
Mounting Effect-only Atoms

### DISCUSSION THREAD

**Discussion by probeiuscorp** [pmndrs/jotai#1838] [Q&A]
Take this rather compact, readable and scalable effect:
```javascript
const userAtom = atom(12925893468)
const pageAtom = atom('/home')

const trackerAtom = atom((get) => {
  const user = get(userAtom)
  const page = get(pageAtom)
  fetch(`/tracker?user=${user}&page=${page}`)
})
```

However, `trackerAtom` will never get mounted. Off the top of my head, you could stick a dummy `useAtom` to load it, or you could replace it with a `useEffect`, but neither feels quite as clean (or as scalable) since this effect has nothing at all to do with the view. Moreover, you could ideally be able to implement this effect without sticking it into the `write`s of the other atoms so that the effect could be completely self-contained.

Is there a way to have this cake and eat it too? To my understanding, there is no way to do this. I would love for something like this to be possible.

**Comment by t6adev** [comment]
Ah, yes. Even if userAtom or pageAtom is mounted, trackerAtom is never mounted.
As you mentioned, we have to mount it in components.
